examples/basic_api_usage.py

```diff
@@ -0,0 +1,23 @@
+"""This is an exmaple of how to use the simple sqlfluff api."""
+
+import sqlfluff
+
+#  -------- LINTING ----------
+
+my_bad_query = "SeLEct  *, 1, blah as  fOO  from myTable"
+
+# Lint the given string and get a list of violations found.
+result = sqlfluff.lint(my_bad_query, dialect="bigquery")
+
+# result =
+# [
+#     {"code": "L010", "line_no": 1, "line_pos": 1, "description": "Inconsistent capitalisation of keywords."}
+#     ...
+# ]
+
+#  -------- FIXING ----------
+
+# Fix the given string and get a string back which has been fixed.
+result = sqlfluff.fix(my_bad_query, dialect="bigquery")
+
+# result = 'SELECT  *, 1, blah AS  foo  FROM mytable\n'

```

Comment:
There are multiple spaces after `SELECT` and `AS`, also multiple spaces before `FROM`. Is this the actual/intended behavior?

Reply:
That's an excellent point! There is currently no linting rule to enforce single whitespace outside of indents and comments, but I think it's a great idea. I'll make an issue for it.

Reply:
See #502

Reply:
wow barry good eyes!